docs/Querying.rst

```diff
@@ -226,6 +226,22 @@ Here we can make our own ``MATCH`` clauses unlike in model scoping.  We have ``w
 **TODO Duplicate this page and link to it from here (or just duplicate it here):**
 https://github.com/neo4jrb/neo4j-core/wiki/Queries
 
+When using strings inside of a ``Query`` method, you have access to an ``identity`` variable that can simplify your code.  For instance:
+
+.. code-block:: ruby
+
+  Student.scope :in_order, -> { order("#{identity}.level_num DESC") }
```

Comment:
As discussed in #1158 I think this probably isn't the best example because you can get away without specify `identity`